tests/sharding-e2e-test.sh

```diff
@@ -24,6 +24,8 @@ set -ex
 echo "Installing createtree..."
 go install github.com/google/trillian/cmd/createtree@latest
 
+echo "Installing cosign..."
+go install github.com/sigstore/cosign/cmd/cosign@latest
```

Comment:
this logic may evolve as time goes on - are we better actually reimplementing the logic here by directly calling the rekor APIs?

Reply:
i think we probably want to test with cosign somewhere, whether it's here or in a prober. not super opinionated on this one.

Reply:
+1 let's remove cosign and use rekor: the problem was reproducible with a SearchLogQuery